web/book/src/reference/syntax/pipes.md

```diff
@@ -1,10 +1,10 @@
 # Pipes
 
-Pipes — the connection between [transforms](../stdlib/transforms/) that make up
-a pipeline — can be either line breaks or a pipe character (`|`).
+Pipes — the connection between [transforms](../stdlib/transforms/) that make up
+a pipeline — can be either linebreaks or a pipe character (`|`).
 
-In almost all situations, line-breaks pipe the result of a line's transform into
-the transform on the following line. For example, the `filter` transform
+In almost all situations, a line break pipe the result of a line's transform
+into the transform on the following line. For example, the `filter` transform
```

Comment:
This doesn't read right. I think the original text was ok or otherwise maybe something like the following:

> In almost all situations, a linebreak (\n) is equivalent to inserting a pipe character (|), which means the relation produced by the current transform is used as the input (last argument) for the transform on the following line.
> 
> For example, the `filter` transform

Reply:
Sorry for the grammar, your proposal is great, I'm adding (you can always add as a suggestion btw!)

Reply:
I actually restructured it slightly...